tests/api.spec.ts

```diff
@@ -367,7 +324,7 @@ describe('Requests to public API', () => {
 
       if (response.value.status === AnalysisStatus.done) {
         expect(response.value.analysisURL.includes(realBundleIdFull)).toBeTruthy();
-        expect(Object.keys(response.value.analysisResults.suggestions).length).toEqual(8);
+        expect(Object.keys(response.value.analysisResults.suggestions).length).toEqual(6);
```

Comment:
question: why did we change this file?

Reply:
@orkamara Because tests were failing on master. Check https://snyk.slack.com/archives/C0135Q8LP61/p1615978344024300